src/frontend/src/components/Posts/LoadAutoScroll.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Container, Button, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  content: {
+    '& > *': {
+      padding: theme.spacing(2),
+      bottom: theme.spacing(4),
+    },
+  },
+}));
+
+function LoadMoreButton({ onScroll }) {
+  const classes = useStyles();
+  const $buttonRef = React.useRef(null);
+
+  // This will make the automatic infinite scrolling feature
+  // Once the "button" is on the viewport(shown on the window),
+  // The new posts are updated(call onClick() -- setSize(size + 1) in Posts.jsx --)
+  React.useEffect(() => {
+    let options = {
+      root: null,
+      rootMargin: '0px',
+      threshold: 1.0,
+    };
+
+    let observer = new IntersectionObserver((entries, observer) => {
```

Comment:
You're not using `observer` arg here, let's remove.  Also, you don't need a block, just do `=> entries.forEach(...)`